google/auth/_helpers.py

```diff
@@ -162,10 +162,10 @@ def update_query(url, params, remove=None):
     # Update the query parameters with the new parameters.
     query_params.update(params)
     # Remove any values specified in remove.
-    query_params = {
+    query_params = dict({
```

Comment:
Apparently `{key: value for item in generator}` is a generator expression as well. So, coverage was complaining about these never actually being executed.